legion/engine/rendering/data/particlepolicies/renderpolicy.cpp

```diff
@@ -0,0 +1,26 @@
+#include <rendering/rendering.hpp>
+#include <rendering/data/particlepolicies/renderpolicy.hpp>
+
+namespace legion::rendering
+{
+    void rendering_policy::OnSetup(particle_emitter& emitter)
+    {
+        emitter.create_uniform<mesh_filter>("mesh_filter", mesh_filter(model.get_mesh()));
+        emitter.create_uniform<gfx::mesh_renderer>("renderer", gfx::mesh_renderer(material, model));
+    }
+
+    void rendering_policy::OnInit(particle_emitter& emitter, size_type start, size_type end)
+    {
+
+    }
+
+    void rendering_policy::OnUpdate(particle_emitter& emitter, float deltaTime, size_type count)
+    {
+
+    }
+
+    void rendering_policy::OnDestroy(particle_emitter& emitter, size_type start, size_type end)
+    {
+
+    }
```

Comment:
if we don't use them, are they necessary?

Reply:
it won't compile if they aren't defined